pkg/api/controller.go

```
@@ -188,13 +188,13 @@ func (c *Controller) Run(reloadCtx context.Context) error {
 
 			caCert, err := os.ReadFile(c.Config.HTTP.TLS.CACert)
 			if err != nil {
-				panic(err)
+				c.Log.Panic().Err(err).Str("caCert", c.Config.HTTP.TLS.CACert).Msg("failed to read file")
```

Can we return the error instead?